muoi07052001/near-app-vote-sc
# Add Event Log

## What
- [x] Implement NEAR's Standard Event Log

## Why
- For the backend to be able to receive actions from Smart Contract

## How
- Implement NEP-297
- Docs: https://nomicon.io/Standards/EventsFormat

## How to QA
- Create/Update something -> Check log on [NEAR Explorer](https://explorer.testnet.near.org/)
